### DISCUSSION TITLE
cant get the build.gradel to work

### DISCUSSION THREAD

**Discussion by YouBetterCat** [Minestom/Minestom#1960] [Help]
plugins {
    id 'java'
}

group = 'de.YouBetterCat'
version = '1.0-SNAPSHOT'

repositories {
    mavenCentral()
    maven { url 'https://jitpack.io' }
}

dependencies {
    testImplementation platform('org.junit:junit-bom:5.9.1')
    testImplementation ('org.junit.jupiter:junit-jupiter')
    implementation ('com.github.Minestom:Minestom:2cdb3911b0')
}

test {
    useJUnitPlatform()
}

--------------------------------------------------------------------------------------------------------------

this is my current build.gradle file but i cant get it to work i get this error message: 
--------------------------------------------------------------------------------------------------------------
MInestomServer2:main: Could not find com.github.Minestom:Minestom:2cdb3911b0.
Searched in the following locations:
  - https://repo.maven.apache.org/maven2/com/github/Minestom/Minestom/2cdb3911b0/Minestom-2cdb3911b0.pom
  - https://jitpack.io/com/github/Minestom/Minestom/2cdb3911b0/Minestom-2cdb3911b0.pom
Required by:
    project :

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html
--------------------------------------------------------------------------------------------------------------

idk if im to stupid to see the mistake that i probably made so can would someone be so kind as to helop me? ty in advance

**Comment by josemoncab** [comment]
Hi, try changing the version (your 2cdb3911b0) to "-SNAPSHOT". I search your version in jitpack and it fails in the build (thats what jitpack says)

**Reply by YouBetterCat** [reply]
hi iv tried what u told me to do, i get this error in the console, 

Execution failed for task ':compileJava'.
> Could not resolve all files for configuration ':compileClasspath'.
   > Could not find com.github.Minestom:Minestom:-SNAPSHOT.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/com/github/Minestom/Minestom/-SNAPSHOT/maven-metadata.xml
       - https://repo.maven.apache.org/maven2/com/github/Minestom/Minestom/-SNAPSHOT/Minestom--SNAPSHOT.pom
       - https://repo.spongepowered.org/maven/com/github/Minestom/Minestom/-SNAPSHOT/maven-metadata.xml
       - https://repo.spongepowered.org/maven/com/github/Minestom/Minestom/-SNAPSHOT/Minestom--SNAPSHOT.pom
       - https://jitpack.io/com/github/Minestom/Minestom/-SNAPSHOT/maven-metadata.xml
       - https://jitpack.io/com/github/Minestom/Minestom/-SNAPSHOT/Minestom--SNAPSHOT.pom
     Required by:
         project :

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html


also this it what the dependencies in the gradle tab look like 
![image](https://github.com/Minestom/Minestom/assets/94190372/e70abf06-4670-440f-af29-f33c43c85b5a)
